lib/.gitignore

```diff
@@ -0,0 +1,5 @@
+pkgconfig/
+*.so
+*.so.1.1
+*.so.1
+*.so.1.0
```

Comment:
Should we make those 1.1, 1.0, 1 a * to avoid revisiting when they become a 2, or a 3?